samples/react/src/config/storybookContent.json

```diff
@@ -1,4 +1,23 @@
 {
```

Comment:
[suggestion] This file could export a JS object rather than JSON. Only reason I'm suggesting this is because JS object syntax is easier to write than JSON syntax. Also, could this data be co-located with the Color Swatches story so it's easier to find?

Reply:
Yes, yes, a thousand times yes!